src/elixir/data_sanitiser/test/date_utils_test.exs

```diff
@@ -0,0 +1,102 @@
+defmodule DataSanitiser.DateUtilsTests do
+  defmodule NormaliseDayTest do
+    use ExUnit.Case
+
+    import DataSanitiser.DateUtils, only: [ normalise_day: 1 ]
+
+    test "An empty string normalises to :nil" do
+      assert normalise_day("") == :nil
+    end
+
+    test "A non empty string normalises to its integer representation" do
+      assert normalise_day("1") == 1
+      assert normalise_day("04") == 4
+      assert normalise_day("31") == 31
+    end
+  end
+
+  defmodule NormaliseMonthTest do
+    use ExUnit.Case
+
+    import DataSanitiser.DateUtils, only: [ normalise_month: 1 ]
+
+    test "Short month names normalise to their integer representation" do
```

Comment:
Yeah, I considered adding it twice but thought it wasn't really a 'shortened' version of the name